src/prefect/environments/storage/local.py

```diff
@@ -95,14 +95,18 @@ def get_flow(self, flow_location: str = None) -> "Flow":
             raise ValueError("No flow location provided")
 
         # check if the path given is a file path
-        if os.path.isfile(flow_location):
-            if self.stored_as_script:
-                return extract_flow_from_file(file_path=flow_location)
+        try:
+            if os.path.isfile(flow_location):
+                if self.stored_as_script:
+                    return extract_flow_from_file(file_path=flow_location)
+                else:
+                    return prefect.core.flow.Flow.load(flow_location)
+            # otherwise the path is given in the module format
             else:
-                return prefect.core.flow.Flow.load(flow_location)
-        # otherwise the path is given in the module format
-        else:
-            return extract_flow_from_module(module_str=flow_location)
+                return extract_flow_from_module(module_str=flow_location)
+        except Exception as exc:
+            self.logger.exception(f"Failed to load Flow from {flow_location}")
+            raise exc
```

Comment:
```suggestion
        except Exception:
            self.logger.exception(f"Failed to load Flow from {flow_location}")
            raise
```

Small tweak to use `raise` not `raise exc` (which will add a chained exception). Slightly cleaner tracebacks.